lokinet_lib/android/src/main/kotlin/io/oxen/lokinet_lib/LokinetLibPlugin.kt

```diff
@@ -140,7 +150,7 @@ class LokinetLibPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
                     result.success(false)
                 }
             }
-            "getStatus" -> {
```

Comment:
why does this need to change name?

Reply:
it doesn't if I rename the other one to "connected"